### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by ExtraShotLatte**
Add final text to end of UG!

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -486,6 +486,15 @@ Paidlancers data are saved in the hard disk automatically on command issue. Ther
 |   [View Upcoming Events](#view-upcoming-events)    |                                 `remind DAYS`                                 |                                                      `remind 2`                                                       |
 |              [Save data](#save-data)               |                                      NIL                                      |                                                          NIL                                                          |
 
+Thank you for taking the time to read through this user guide:smile:! We hope that it has provided you with the
+information you need to make the most out of Paidlancers!:innocent:
```
Comment (glozxi) [inline docs/UserGuide.md:490]: Would it look nicer to add a space before every emoji?

**Review by glozxi — CHANGES_REQUESTED** [review]
Very nice, just a nit

File: docs/UserGuide.md (hunk 1 above)
Comment (ExtraShotLatte) [reply docs/UserGuide.md]: Fixed!

**Review by glozxi — APPROVED** [review]
LGTM

**Pull request merged**